livekit-plugins/livekit-plugins-anthropic/livekit/plugins/anthropic/llm.py

```diff
@@ -61,6 +63,8 @@ class LLMOptions:
     temperature: float | None
     parallel_tool_calls: bool | None
     tool_choice: Union[ToolChoice, Literal["auto", "required", "none"]] | None
+    enable_caching: bool | None = None
```

Comment:
Maybe to be future proof, something like this
```suggestion
    caching: Literal["ephemeral"] | None = None
```